content/providers/releases.md

```diff
@@ -3,13 +3,67 @@ title: "Provider releases"
 weight: 50
 ---
 
-The below release notes are for the Upbound Official providers. The notes here
-only contain noteworthy changes and you should refer to each release's GitHub
```

Comment:
I made this change due to the following error:
<img width="1190" alt="Screenshot 2023-08-02 at 15 07 12" src="https://github.com/upbound/docs/assets/103541666/1b411f1b-7d35-44ee-a2c7-3d32b94f309d">